pkg/recordset/recordset.go

```diff
@@ -204,10 +262,10 @@ func validStackInstallationTag(stacks *cloudformation.DescribeStacksOutput, inst
 
 func (m *Manager) createMissingTargetStacks(sourceStacks, targetStacks []cloudformation.Stack) error {
```

Comment:
Just for me to understand. What are `sourceStacks` and `targetStacks`?

Reply:
`sourceStacks` is the Cloudformation stack representing the guest cluster which is created by `aws-operator`.
`targetStacks` is the Cloudformation created by `route53-manager` which holds the DNS/Route53 configuration.